source/Cosmos.Core_Plugs/Cosmos.Core_Plugs.csproj

```diff
@@ -22,4 +22,8 @@
     <Folder Include="Properties\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="System\Threading\TaskImpl.cs" />
```

Comment:
Why is it included as None?

Reply:
I think this has been done automatically by Visual Studio, where should it be?

Reply:
It shouldn't be added at all.